conf/luna.conf

```diff
@@ -63,7 +63,7 @@ HomeDoubleClickDuration=150
 
 [Memory]
 CardLimit=-1
-AppsToAllowInLowMemory=com.palm.app.phone;com.palm.app.contacts;com.palm.app.messaging
+AppsToAllowInLowMemory=com.palm.app.phone;com.palm.app.contacts;com.palm.app.messaging;org.webosports.app.phone;org.webosports.app.contacts;org.webosports.app.messaging
```

Comment:
I guess we could drop them at some point. If not now.. They anyway don't exist on our target, idea was that you could theoretically copy them over from 3.0.x ;)